benchmarks/bayesmark/report_template.md

```diff
@@ -0,0 +1,36 @@
+# Benchmark Result Report
+
+* Number of Solvers: {num_solvers}
+* Number of Models: {num_models}
+* Number of Datasets: {num_datasets}
+* Number of Problems: {num_problems}
+
+Final score for each problem is calculated as `100 x (1-loss)`. Solver with lowest score in each problem wins. For more details visit [bayesmark docs.](https://bayesmark.readthedocs.io/en/stable/scoring.html)
```

Comment:
Given a solver with loss = 0 (good) and other with loss = 1. Scores become 100 * (1-0) = 100 and 100 * (1-1) = 0 respectively. If `Solver with lowest score in each problem wins`, the latter solver (loss = 1) wins. But it doesn't make sense, I think.

Reply:
I think you're right. In our case `loss` is normed CV score returned by last objective function evaluation, averaged across runs. Since we are using MSE for regression and negative log-likelihood for classification, which are both minimized, behavior you're describing would be correct (highest score wins). CC @HideakiImamura

Reply:
Done in c08bcacd67e3c3cb22338f34aa688f29e8676712.